$NetBSD: patch-aa,v 1.4 2009/01/19 22:54:24 he Exp $

Avoid Makefile.PL querying stdin, possibly causing a build to stall.

--- Makefile.PL.orig	2009-01-18 16:35:38.000000000 +0100
+++ Makefile.PL
@@ -17,9 +17,9 @@ WARNING! You seem to have an older versi
 See the README for details.
 
 END_OF_WARNING
-print "Press enter to continue ... ";
-my $x = <STDIN>;
-last if $x;
+# print "Press enter to continue ... ";
+# my $x = <STDIN>;
+# last if $x;
   }
 };
 
